rubin_scheduler/scheduler/example/example_scheduler.py

```diff
@@ -1133,20 +1141,20 @@ def ddf_surveys(detailers=None, season_unobs_frac=0.2, euclid_detailers=None, ns
     return [survey1, survey2]
 
 
-def ecliptic_target(nside=SURVEY_START_MJD, dist_to_eclip=40.0, dec_max=30.0, mask=None):
+def ecliptic_target(nside=DEFAULT_NSIDE, dist_to_eclip=40.0, dec_max=30.0, mask=None):
     """Generate a target_map for the area around the ecliptic
 
     Parameters
     ----------
-    nside : int (DEFAULT_NSIDE)
+    nside : int
```

Comment:
At some point some formatting work is needed in rubin_scheduler to fix the markup so the auto-generated docs have working links, for example by putting backticks around the types for parameters.
I'm not convinced that in this PR is the right place to do this, though, but we should do this eventually.